lib/internal/checkUrl.js

```diff
@@ -9,9 +9,9 @@ var bhttp = require("bhttp");
 /*
 	Checks a URL to see if it's broken or not.
 */
-function checkUrl(link, baseUrl, options, callback)
+function checkUrl(link, baseUrl, options, callback, cache)
```

Comment:
I think `cache` should go before `callback`, since the callback is really the final step of the function